### PR TITLE
*: fix descriptions and update format

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Fix some wording and format issues brought by #3278 and #3166 
- Overview
- TiKV thread pools

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): #3783 @husiyu FYI.

### CODE REVIEW

**Comment by lilin90** [comment]
@Little-Wallace PTAL

File: overview.md [hunk 1]
```diff
@@ -33,17 +33,17 @@ TiDB 是 PingCAP 公司自主设计、研发的开源分布式关系型数据库
 ## 四大核心应用场景
 
 - 对数据一致性及高可靠、系统高可用、可扩展性、容灾要求较高的金融行业属性的场景
-   
-   众所周知，金融行业对数据一致性及高可靠、系统高可用、可扩展性、容灾要求较高，传统的解决方案是同城两个机房提供服务、异地一个机房提供数据容灾能务但不提供服务，此解决方案存在资源利用率低、维扩成本高、RTO 及 PRO 无法真实达到企业所期望的值。TiDB 采用多副本 + Mutli Raft 协议的方式将数据调度到不同的机房、机架、机器，当部分机器故障时系统可自动进行切换，确保系统的 RTO <= 30s 及 RPO = 0 。
-       
-- 对存储容量、可扩展性、并发要求较高的海量数量及高并的 OLTP 场景
-    
+
+    众所周知，金融行业对数据一致性及高可靠、系统高可用、可扩展性、容灾要求较高。传统的解决方案是同城两个机房提供服务、异地一个机房提供数据容灾能务但不提供服务，此解决方案存在以下缺点：资源利用率低、维护成本高、RTO 及 RPO 无法真实达到企业所期望的值。TiDB 采用多副本 + Multi-Raft 协议的方式将数据调度到不同的机房、机架、机器，当部分机器出现故障时系统可自动进行切换，确保系统的 RTO <= 30s 及 RPO = 0 。
```
Comment (lilin90) [inline overview.md:37]:
```suggestion
    众所周知，金融行业对数据一致性及高可靠、系统高可用、可扩展性、容灾要求较高。传统的解决方案是同城两个机房提供服务、异地一个机房提供数据容灾能务但不提供服务，此解决方案存在以下缺点：资源利用率低、维护成本高、RTO (Recovery Time Objective) 及 RPO (Recovery Point Objective) 无法真实达到企业所期望的值。TiDB 采用多副本 + Multi-Raft 协议的方式将数据调度到不同的机房、机架、机器，当部分机器出现故障时系统可自动进行切换，确保系统的 RTO <= 30s 及 RPO = 0 。
```

File: overview.md [hunk 2]
```diff
@@ -33,17 +33,17 @@ TiDB 是 PingCAP 公司自主设计、研发的开源分布式关系型数据库
 ## 四大核心应用场景
 
 - 对数据一致性及高可靠、系统高可用、可扩展性、容灾要求较高的金融行业属性的场景
-   
-   众所周知，金融行业对数据一致性及高可靠、系统高可用、可扩展性、容灾要求较高，传统的解决方案是同城两个机房提供服务、异地一个机房提供数据容灾能务但不提供服务，此解决方案存在资源利用率低、维扩成本高、RTO 及 PRO 无法真实达到企业所期望的值。TiDB 采用多副本 + Mutli Raft 协议的方式将数据调度到不同的机房、机架、机器，当部分机器故障时系统可自动进行切换，确保系统的 RTO <= 30s 及 RPO = 0 。
-       
-- 对存储容量、可扩展性、并发要求较高的海量数量及高并的 OLTP 场景
-    
+
+    众所周知，金融行业对数据一致性及高可靠、系统高可用、可扩展性、容灾要求较高。传统的解决方案是同城两个机房提供服务、异地一个机房提供数据容灾能务但不提供服务，此解决方案存在以下缺点：资源利用率低、维护成本高、RTO (Recovery Time Objective) 及 RPO (Recovery Point Objective) 无法真实达到企业所期望的值。TiDB 采用多副本 + Multi-Raft 协议的方式将数据调度到不同的机房、机架、机器，当部分机器出现故障时系统可自动进行切换，确保系统的 RTO <= 30s 及 RPO = 0 。
```
Comment (scsldb) [inline overview.md:37]: 能务 改成 能力

Comment (lilin90) [reply overview.md:37]: Fixed in #3818.

**Pull request merged**